### DISCUSSION TITLE
How to use the "timestamp" with discord webhook (/admin/webhooks)

### DISCUSSION THREAD

**Discussion by Ik1497** [getfider/fider#1099] [Q&A] [answered]
I've tried the default `{{ .post_created_at }}`, but it came out in the wrong format. I also tried the format that discord uses: `{{ format "2022-10-23T15:00:00.000Z" .post_created_at }}`, but that doesn't do it. Any help would be highly appreciated 🙏

**Comment by indyteo** [comment, marked as answer]
Hi! According to the [date format documentation in Go](https://yourbasic.org/golang/format-parse-string-time-date-example/#standard-time-and-date-formats) (referenced in the Fider Webhooks documentation of the `format` function [here](https://fider.io/docs/using-webhooks)), I think the template you're looking for is: `2006-01-02T15:04:05-0700` (note that the values are numbers, but they must not be modified, `2006` cannot be replaced by `2022` in the template).

**Reply by Ik1497** [reply]
Yes that works, thank you so much!
